.golangci.yaml

```diff
@@ -19,3 +20,4 @@ issues:
     - unused # getting many unused false positives in neo4j backend
     - forcetypeassert # this backend does this a lot
   max-same-issues: 50
+  new-from-rev: HEAD~1 # only run on changes from last commit
```

Comment:
I believe this doesn't work very well in PRs, and https://github.com/guacsec/guac/blob/main/.github/workflows/ci.yaml#L127-L128 should take care of it.